packages/liveblocks-core/src/room.ts

```diff
@@ -832,6 +850,12 @@ type PrivateRoomApi = {
     explicitClose(event: IWebSocketCloseEvent): void;
     rawSend(data: string): void;
   };
+
+  // Used to store metadata related to comments
+  comments: {
+    lastRequestedAt: Date | null; // Stores the timestamp when threads and notifications were last requested for the room
+    readonly queries: Set<string>; // Stores queries that have been made in the room
```

Comment:
The reason I have stored these information inside a room context is because they are only relevant as long as the room is alive (i.e., not destroyed). For instance, the set `queries` stores query keys for `getThread()` and `getRoomNotificationSettings` queries that have been made. 

I could have stored this as a map inside `room.tsx` where key would be `roomId` and value would be this metadata information. But, I realized there is a new layer of complexity with doing that, especially because a client can have a room with the same id rendered at multiple places (via `RoomProvider`), and these information (about `lastRequestedAt`, `queries`) are expected to be destroyed when the room is destroyed, so we would need to keep track of how many rooms for the same room id is being rendered currently and so on.

Reply:
Can you clarify why we have to clear this information when the room is destroyed? We're not clearing the cache when the room is destroyed, so I guess we don't have to do it in this case either?

Unrelated to this PR, but even the code in `store.ts` should be moved to `@liveblocks/react` ultimately. It was at some point, but I think it was moved to `@liveblocks/client` when we introduced the hidden symbol field on the client. I want to move all that code to `@liveblocks/react` right after we release the `1.10`